source/install/desktop.rst

```diff
@@ -117,72 +119,71 @@ A snap is available for systems that have Snapcraft installed. Snapcraft is inst
 
   ``sudo snap install mattermost-desktop --beta``
 
-2. To run Mattermost, open **Dash** (located at top left corner) and input ``mattermost``, then click the Mattermost icon.
+2. To run Mattermost, open **Dash** (located at the top left corner), enter **mattermost**`, then click the Mattermost icon.
 
 Troubleshooting
---------------------------------------------------
+-----------------
 
 "Installation has failed" dialog
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    
-    The app data might be corrupted - remove all the files in ``%LOCALAPPDATA%\mattermost``, then try re-installing the app.
+
+The app data might be corrupted. Remove all the files in ``%LOCALAPPDATA%\mattermost``, then try reinstalling the app.
     
 "The application "Mattermost" can't be opened" dialog
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   On macOS Catalina, this dialog can be triggered if the Mac Archive Utility is the default method for decompressing files. In this case using a third-party tool such as [Keka](https://www.keka.io/) or [Unarchiver](https://macpaw.com/the-unarchiver) may resolve the problem.
+On macOS Catalina, this dialog can be triggered if the Mac Archive Utility is the default method for decompressing files. In this case using a third-party tool such as [Keka](https://www.keka.io/) or [Unarchiver](https://macpaw.com/the-unarchiver) may resolve the problem.
```

Comment:
The links are not formatted correctly.

Reply:
Yikes, thanks for catching that! Sorted. :)

Reply:
This one still reproduces.